packages/mdc-toolbar/README.md

```diff
@@ -136,6 +230,82 @@ The provided modifiers are:
 | `mdc-toolbar__section--align-start`  | Makes section aligns to the start.      |
 | `mdc-toolbar__section--align-end`    | Makes section aligns to the end.        |
+
+
+## JS Usage
+
+### Including in code
+
+#### ES2015
+
+```javascript
+import {MDCToolbar, MDCToolbarFoundation} from 'mdc-toolbar';
+```
+
+#### CommonJS
+
+```javascript
+const mdcToolbar = require('mdc-toolbar');
+const MDCToolbar = mdcToolbar.MDCToolbar;
+const MDCToolbarFoundation = mdcToolbar.MDCToolbarFoundation;
+```
+
+#### AMD
+
+```javascript
+require(['path/to/mdc-toolbar'], mdcToolbar => {
+  const MDCToolbar = mdcToolbar.MDCToolbar;
+  const MDCToolbarFoundation = mdcToolbar.MDCToolbarFoundation;
+});
+```
+
+#### Global
+
+```javascript
+const MDCToolbar = mdc.toolbar.MDCToolbar;
+const MDCToolbarFoundation = mdc.toolbar.MDCToolbarFoundation;
+```
+
+### Instantiation
+
+```javascript
+import {MDCToolbar} from 'mdc-toolbar';
+
+const toolbar = new MDCToolbar(document.querySelector('.mdc-toolbar'));
+```
+
+### Event
+
+| Event Name | Event Data Structure | Description |
+| --- | --- | --- |
+| `change` | {detail: number} | Emitted the ratio of current flexible space to total flexible space height. So when it is minimized, ratio equals to 0 and when it is maximized, ratio equals to 1. |
+
+
+### Adapter
+
+| Method Signature | Description |
+| --- | --- |
+| `hasClass(className: string) => void` | Check if the root element of the component has the given className. |
+| `addClass(className: string) => void` | Adds a class to the root element of the component. |
+| `removeClass(className: string) => void` | Removes a class from the root element of the component. |
+| `registerScrollHandler(handler: Function) => void` | Registers a handler to be called when user scrolls. Our default implementation adds the handler as a listener to the window's `scroll` event. |
+| `deregisterScrollHandler(handler: Function) => void` | Unregisters a handler to be called when user scrolls. Our default implementation removes the handler as a listener to the window's `scroll` event. |
+| `registerResizeHandler(handler: Function) => void` | Registers a handler to be called when the surface (or its viewport) resizes. Our default implementation adds the handler as a listener to the window's `resize` event. |
+| `deregisterResizeHandler(handler: Function) => void` | Unregisters a handler to be called when the surface (or its viewport) resizes. Our default implementation removes the handler as a listener to the window's `resize` event. |
+| `getViewportWidth() => number` | Get viewport (window) width. |
+| `getViewportScrollY() => number` | Get the number of pixels that the content of body is scrolled upward.|
+| `getOffsetHeight() => number` | Get root element `mdc-toolbar` offsetHeight. |
+| `getFlexibleRowElementOffsetHeight() => number` | Get flexible row element offsetHeight. |
+| `notifyChange() => evtData` | Broadcasts an event with the remaining ratio of flexible space. |
```

Comment:
I think the method signature here is `notifyChange(evtData: {flexibleExpansionRatio: number}) => void`